cmd/snap/cmd_repair_repairs.go

```diff
@@ -0,0 +1,82 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/release"
+)
+
+func runSnapRepair(cmdStr string, args []string) error {
+	snapRepairPath := filepath.Join(dirs.DistroLibExecDir, "snap-repair")
```

Comment:
do weed a comment that we can do this because repairs are core only?  do we want to give some kind of error message on classic about that instead of giving empty lists? not sure myself about the latter

Reply:
Good point, I think we should already error here if we are on classic. Most classic distros will probably not even include snap-repair so its pointless to pass-through to it.

Reply:
I updated the output when this is run on classic, but I'm open for suggestions for better errors.